docs/develop/api-reference/sorted-set-collections.md

```diff
@@ -1,13 +1,13 @@
 ---
 sidebar_position: 4
 sidebar_label: Sorted sets
-title: Sorted set API reference
+title: Sorted set collections API reference
 description: Learn how to interact with the API for sorted set collection data types in Momento Serverless Cache.
 slug: /develop/api-reference/collections/sortedsets
 ---
 
-# Sorted sets collection data types
-A sorted set in Momento Serverless Cache is a collection of unique elements with a value (string) and score (signed double 64-bit float) pair. The elements in the item are ordered by score value.
+# Sorted set collections
+A sorted set in Momento Serverless Cache is a collection of unique elements with a value (string) and score (signed double 64-bit float) pair. The elements in the item are ordered by the score's value.
```

Comment:
Does the value have to be a string?  Also, I think I would find a different way to approach the second sentence. When you talk about a score's value it might create confusion with the value which is the other part of the pair. Maybe just... "The elements in the item are ordered by their scores." ?

Reply:
Fixed